docs/manpage.rst

```diff
@@ -752,6 +752,18 @@ If no node can be selected, the test will be marked as a failure with an appropr
 
       Slurm OR constraints and parenthesized expressions are supported in flexible node allocation.
 
+   .. versionchanged:: 4.7
+      The test is not failed if not enough nodes are available, but it is skipped instead.
```

Comment:
```suggestion
      The test does not fail if not enough nodes are available, but it is skipped instead.
```

Reply:
In this case it is the framework that "fails the test" (marks it as a failure) and not the test that fails.

Reply:
I will change it to "the test is not marked as a failure [...]" to make it clearer.